azure-pipelines.yml

```diff
@@ -96,6 +96,7 @@ jobs:
       tqdm
       threadpoolctl
       fasteners
+      pyedr
```

Comment:
It's not common practice to have optional deps in azure pipeline runners, I'd remove it for now.